packages/core/plopfile.js

```diff
@@ -1,11 +1,11 @@
-const Component = require("./plop/component");
-const Tests = require("./plop/tests");
-const Stories = require("./plop/stories");
-const HooksStory = require("./plop/hooks");
+import component from "./plop/component/index.js";
+import tests from "./plop/tests/index.js";
+import stories from "./plop/stories/index.js";
+import hooksStory from "./plop/hooks/index.js";
```

Comment:
wouldn't the same as it was before still work?

```suggestion
import component from "./plop/component";
import tests from "./plop/tests";
import stories from "./plop/stories";
import hooksStory from "./plop/hooks";
```

Reply:
It's needed because we're out of src and it requires some settings which I didn't want to involve in this pr